Source/MQTTnet/Adapter/MqttConnectingFailedException.cs

```diff
@@ -5,12 +5,13 @@ namespace MQTTnet.Adapter
 {
     public class MqttConnectingFailedException : MqttCommunicationException
     {
-        public MqttConnectingFailedException(MqttClientConnectResultCode resultCode)
-            : base($"Connecting with MQTT server failed ({resultCode.ToString()}).")
+        public MqttConnectingFailedException(MqttClientAuthenticateResult resultCode)
```

Comment:
You already added a new "Result" property. So please name the parameter also "result" because it no longer the result code only.

Reply:
done